config/initializers/scrna_config.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# This hash is used to store constants used in pooling and chip loading calculations for samples in the scRNA pipeline.
+Rails.application.config.scrna_config = {
+  # Maximum volume of a sample in microlitres
+  maximum_sample_volume: 60.0,
+  # Minimum volume of a sample in microlitres
+  minimum_sample_volume: 5.0,
+  # Minimum volume required for resuspension in microlitres
+  minimum_resuspension_volume: 10.0,
+  # Conversion factor from millilitres to microlitres
+  millilitres_to_microlitres: 1_000.0,
+  # Number of cells required for each sample going into the pool
+  required_number_of_cells_per_sample_in_pool: 30_000,
+  # Factor accounting for wastage of material when transferring between labware
+  wastage_factor: 0.95238,
+  # Desired concentration of cells per microlitre for chip loading
+  desired_chip_loading_concentration: 2400,
+  # Desired number of cells per well in the chip
+  desired_number_of_cells_per_chip_well: 90_000,
```

Comment:
is this `number_of_cells_per_chip_well` constant obsolete? There is the `request_metadata` field `cells_per_chip_well` as well.

Reply:
yes, removed references